content/en/logs/guide/correlate-your-full-stack-data.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Correlate your full stack logs, traces and views
+kind: guide
+aliases:
+  - /logs/guide/xxx
+further_reading:
+  - link: /logs/guide/logs-rbac/
+    tag: Documentation
+    text: "Set up Roles Based Access Controls (RBAC) for Log Management"
+  - link: /agent/logs/advanced_log_collection
+    tag: Documentation
+    text: "Filter and Redact logs with Advanced Log Collection"
+  - link: /agent/guide/autodiscovery-management/
+    tag: Documentation
+    text: "Exclude containers from Log Collection with Autodiscovery"
+
+---
+
+## Overview
+
+With [unified service tagging][1] you already have high level correlation capabilities. But sometimes the starting point of your investigation is a single log, trace or view. Correlating them with other data gives context to estimate business impact and find root causes in a few clicks.
+
+{{< img src="logs/guide/correlate-your-full-stack-data/full-stack-cover.png" alt="Full stack correlation" style="width:80%;" >}}
+
+Correlating your logs also eases [aggressive sampling strategy without losing entity-level consistency][2] with attributes like `trace_id` and `session_id`.
+
+This guide walks you through the steps you should take to correlate your full stack logs, traces and view.
+
+At first, [correlating application logs](#correlate-application-logs) offers extensive visibility across your stack, but some very specific use-cases require to prolong the correlation deeper to your stack:
+
+1. [Correlate proxy logs](#correlate-proxy-logs)
+2. [Correlate database logs](#correlate-database-logs)
+3. [Correlate RUM views](#correlate-rum-views)
+4. [Correlate browser logs](#correlate-browser-logs)
+
+At last, [leverage the correlation to troubleshoot Synthetic tests](#leverage-the-correlation-to-troubleshoot-synthetic-tests).
+
+Note: Depending on your use cases, you may skip steps. Steps dependant from others are explicit.
+
+## Set up the correlation
+
+### Correlate application logs
+
+#### Why?
+
+Application logs are the backbone of your context that give most of the code and business logic issues. They can even help you solve other services issues, e.g. most ORMs logs database errors.
+
+#### How?
+
+Use one of the [various OOTB correlations][3]. If you use a custom tracer or if you have any issues, you can go on the [correlation FAQ][4].
+
+### Correlate proxy logs
+
+#### Why?
+
+Proxy logs provide more information than application logs: they cover more entrypoints and give information on static content and redirections.
+
+#### How?
+
+The application tracer generates Trace IDs by default. This can be changed by injecting `x-datadog-trace-id` in HTTP Request headers.
```

Comment:
```suggestion
The application tracer generates trace IDs by default. This can be changed by injecting `x-datadog-trace-id` into HTTP Request headers.
```